includes/Admin/Admin_Page.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Admin\Admin_Page
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Admin;
+
+/**
+ * Class is handling admin tools page functionality.
+ *
+ * @since n.e.x.t
+ */
+class Admin_Page {
+
+	/**
+	 * Initialize hooks.
+	 */
+	public function add_hooks() {
+		add_action( 'admin_menu', array( $this, 'add_page' ) );
+	}
+
+	/**
+	 * Register the admin page under the tools menu.
```

Comment:
See above.
```suggestion
	 * Registers the admin page under the tools menu.
	 *
	 * @since n.e.x.t
```

Please update this for the other class methods as well.

Reply:
I have updated this now, thank you!